docs/source/release_notes/release-notes.rst

```diff
@@ -46,6 +46,15 @@ v0.21.0
            This is useful for keeping batch sizes small if the rows are wide.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/947>`__)
 
+    *    - |improved|
+         - Added a significant amount of logging at the TRACE level, aimed at tracking down the ``MultipleRunningTimestampServicesError``.
+           If clients are hitting this error, then they should add trace logging for the following packages:
+              - ``com.palantir.atlasdb.cassandra``
+              - ``com.palantir.atlasdb.factory``
+              - ``com.palantir.atlasdb.keyvalue.cassandra``
+              - ``com.palantir.timestamp``
```

Comment:
Do we also want a note that some stuff is logged at INFO level (actually, just fetching the TimestampService)